sanitiser/search.js

```diff
@@ -17,10 +17,6 @@ var sanitizeAll = require('../sanitiser/sanitizeAll'),
 
 var sanitize = function(req, cb) { sanitizeAll(req, sanitizers, cb); };
 
-// export sanitize for testing
-module.exports.sanitize = sanitize;
-module.exports.sanitiser_list = sanitizers;
-
```

Comment:
It's no longer required since the old tests are just re-testing the already-tested individual sanitisers.  The new test proxyquire's all the component sanitisers and verifies that they were called and in the correct order.  See https://github.com/pelias/api/blob/2744244f628627a3e84b23d4e39f83ddce23b4dd/test/unit/sanitiser/search.js#L9